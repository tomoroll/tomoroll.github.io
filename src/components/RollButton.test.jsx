import React from 'react';
import renderer from 'react-test-renderer';
import RollButton from './RollButton';

it('renders correctly', () => {
  const tree = renderer
    .create(<RollButton onClick={() => {}} text="text" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
