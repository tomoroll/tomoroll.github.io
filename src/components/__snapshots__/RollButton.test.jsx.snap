// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="btn btn-primary btn-lg col-md-3"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  Roll
</button>
`;
